macos/pkg/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ add_custom_command(
     DEPENDS mozillavpn
     COMMAND ${CMAKE_COMMAND} -E remove_directory -f ${CMAKE_CURRENT_BINARY_DIR}/staging
     COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_CURRENT_BINARY_DIR}/staging
-    COMMAND ${CMAKE_COMMAND} -E copy_directory $<TARGET_BUNDLE_DIR:mozillavpn>
```

Comment:
This change surprises me because I thought the `mozillavpn` target was a bundle (ie: we set the `MACOSX_BUNDLE` target). Could you confirm that we get the same directory structure built by the `pkg` target after making this change as we used to get with our old qmake builds after making this change?

Reply:
Yes, I think it is the same directory structure. This is where pkg was located: `.../mozilla-vpn-client/build/macos/pkg/staging/MozillaVPN.pkg`, is that as expected?

I was unable to install though, here is the error I got:

<img width="275" alt="Screenshot 2022-07-22 at 11 48 19" src="https://user-images.githubusercontent.com/25176023/180414043-8e848c8f-0371-4707-b10d-9c99c6e19289.png">

Do you think this is related? If not maybe I can go ahead and merge this one and file a bug to look into the other one.

Reply:
Yeah, I think there's a problem with this change to the installer package. I gave it a test and I found that the installer package only contains the VPN binary and is missing the rest of the bundled contents. After building the installer package, you should be able to take a look in `<builddir>/macos/pkg/staging/Applications/` and there should be a directory named `Mozilla VPN.app` which has the same directory structure as in `<builddir>/src/Mozilla VPN.app`